### PR TITLE
Update ServiceAccounts

### PR DESCRIPTION
**Pull request by samblackk**
Attached to https://github.com/astronomer/issues/issues/349

Updates Orbit to use the new queries/mutations for Service Accounts.

### CODE REVIEW

File: src/modules/service-accounts/Configure/index.js [hunk 1]
```diff
@@ -86,4 +88,4 @@ ConfigureServiceAccount.propTypes = {
   apiKey: PropTypes.string,
 }
 
-export default Data(ConfigureServiceAccount)
+export default withRouter(Data(ConfigureServiceAccount))
```
Comment (ashb) [inline src/modules/service-accounts/Configure/index.js]: What's this for?

File: src/modules/service-accounts/Data/Create.js [hunk 2]
```diff
@@ -3,21 +3,33 @@ import React from 'react'
...
-    const variables = getVars({ deploymentId: props.deploymentId, getData })
+    const variables = {
+      deploymentUuid: props.deploymentId,
+      workspaceUuid: (getData && getData.workspaceId) || undefined,
+    }
+
     const query = {
-      name: api.ServiceAccounts,
+      name: isWorkspace(variables)
+        ? api.WorkspaceServiceAccount
```
Comment (ashb) [inline src/modules/service-accounts/Data/Create.js]: This was plural (`ServiceAccounts`) but singular now. Typo or intentional?

File: src/modules/service-accounts/Configure/index.js (hunk 1 above)
Comment (samblackk) [reply src/modules/service-accounts/Configure/index.js]: The get the router props - we're using `match` to get the `serviceAccount` id out of the url.

File: src/modules/service-accounts/Data/Create.js (hunk 2 above)
Comment (samblackk) [reply src/modules/service-accounts/Data/Create.js]: Ah - good catch - initially I had this as singular as it's the return refetch query, but then decided to leave the current methods as they were and didn't come back and update this. Fixed up now!

**Pull request merged**